public/javascripts/mce_editor.js

```diff
@@ -74,6 +74,14 @@ tinyMCE.init({
     // As of version 5.0, TinyMCE has a very limited mobile version it inflicts on Android and iOS phones by default.
     // This forces the desktop version. Note that this is only implicitly documented and may break.
     theme: "silver",
+  },
+
+  setup: function (editor) {
+    // Update character count when switching to and editing in TinyMCE
+    editor.on('init change undo redo', function() {
+      editor.save();
+      $j(editor.targetElm).trigger('change');
```

Comment:
`targetElm` appears to be an undocumented property for TinyMCE, is it meant to be [the textarea that gets replaced by an editor instance](https://www.tiny.cloud/docs/tinymce/5/apis/tinymce.editor/#getElement), or [the container element of the editor](https://www.tiny.cloud/docs/tinymce/5/apis/tinymce.editor/#getContainer), or something else? If we can, let's use one of the methods on the official editor API instead.